src/python/pants/backend/python/subsystems/python_native_code.py

```diff
@@ -127,47 +125,18 @@ def check_build_for_current_platform_only(self, targets):
         raise IncompatiblePlatformsError(
             dedent(
                 """\
-                Pants doesn't currently support cross-compiling native code.
-                The following targets set platforms arguments other than ['current'], which is unsupported for this reason.
-                Please either remove the platforms argument from these targets, or set them to exactly ['current'].
-                Bad targets:
-                {}
-                """.format(
+      Pants doesn't currently support cross-compiling native code.
+      The following targets set platforms arguments other than ['current'], which is unsupported for this reason.
+      Please either remove the platforms argument from these targets, or set them to exactly ['current'].
+      Bad targets:
+      {}
+      """.format(
```

Comment:
Bad change. Might not actually matter, but this is production code so it would be good to revert to the prior formatting.

Reply:
Fixed.